web/spacewalk-web.changes.deneb-alpha.webui_bump_beta2_uyuni202403

```diff
@@ -0,0 +1,2 @@
+- Bump the SUSE Manager WebUI version to 5.0.0 Beta2
```

Comment:
Don't w have more changes for `spacewalk-web`? If so, there's no need for more changelog entries and we avoid confusion for SUSE Manager and Uyuni users so they see stuff that won't really affect them.

Reply:
yes, good point.  let me remove  the  changelog related entry